pkg/operation/care/constraints.go

```diff
@@ -207,21 +219,23 @@ func IsProblematicWebhook(
 	rules []admissionregistrationv1beta1.RuleWithOperations,
 	timeoutSeconds *int32,
 ) bool {
+	if timeoutSeconds != nil && *timeoutSeconds > WebhookMaximumTimeoutSecondsNotProblematic {
```

Comment:
This change will mark all webhooks with a high timeout as problematic and block hibernation, even if the webhooks are not even acting on critical resources / on the `kube-system` namespace.